kubernetes/samples/charts/util/setup.sh

```diff
@@ -130,12 +154,12 @@ function deleteVoyager() {
 }
 
 function deleteTraefik() {
-  if [ "$(helm list | grep $TNAME |  wc -l)" = 1 ]; then
-    echo "Delete Traefik operator." 
-    helm delete --purge $TNAME
-    kubectl delete ns traefik
+  if [ "$(helm list ${t_list_args}| grep $TNAME |  wc -l)" = 1 ]; then
+    echo "Uninstall Traefik operator." 
+    helm uninstall $TNAME ${t_uninstall_args}
+    kubectl delete ns ${TSPACE}
   else
-    echo "Traefik operator has already been deleted." 
+    echo "Traefik operator has already been unistalled" 
```

Comment:
uninstalled

Reply:
Now using delete command instead of uninstall.  Uninstall does not work in 2.x version
3.x -  helm delete --keep-history 
2.x -  helm delete --purge   ( purge not supported )